src/pages/contactUs/components/TitleBlock.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const SubTitle = styled.div`
+  color: ${({ theme }) => theme.colors.greyLight};
+  font-size: 12px;
+  text-transform: uppercase;
+  letter-spacing: 2px;
+`
+const Title = styled.div`
+  color: ${({ theme }) => theme.colors.blackLight};
+  font-size: 26px;
+  font-weight: 600;
+  padding-top: 5px;
+  text-transform: uppercase;
+`
+
+interface PropsTitleBlock {
+  title: string
+}
+
+export const TitleBlock: React.FC<PropsTitleBlock> = ({ title }) => {
```

Comment:
What is the difference between `TitleBlock` and `TitleBar` components?

Reply:
![Annotation 2020-06-15 113607](https://user-images.githubusercontent.com/33199975/84642769-378a7080-aefd-11ea-84a7-a5770db62d2e.png)